back/sssscs/src/main/java/com/ib/user/UserService.java

```diff
@@ -103,6 +105,63 @@ public boolean isTimeFor2FA(User user) {
 		LocalDateTime now = LocalDateTime.now();
 		
 		long minutesPassed = ChronoUnit.MINUTES.between(lastTime, now);
-		return minutesPassed > 5;
+		return minutesPassed >= 5;
```

Comment:
Extract `5` to static final field

Reply:
[dde0be7](https://github.com/magley/sssscs/pull/39/commits/dde0be784ddd5c55a68fed3e4d3b9842a99d2f51)